docs/methods/cardanoGetPublicKey.md

```diff
@@ -1,72 +1,55 @@
 ## Cardano: get public key
-Retrieves [BIP32-Ed25519](https://cardanolaunch.com/assets/Ed25519_BIP.pdf) extended public derived by given [BIP32-Ed25519](https://cardanolaunch.com/assets/Ed25519_BIP.pdf) path.
-User is presented with a description of the requested key and asked to confirm the export on Trezor.
+Retrieve a BIP32-Ed25519 extended public derived by given BIP32-Ed25519 path.
 
 ES6
 ```javascript
 const result = await TrezorConnect.cardanoGetPublicKey(params);
 ```
 
-CommonJS
-```javascript
-TrezorConnect.cardanoGetPublicKey(params).then(function(result) {
-
-});
-```
 
 ### Params
-[****Optional common params****](commonParams.md)
+
```

Comment:
Is there a reason to remove references to commonParams (https://github.com/trezor/connect/blob/develop/docs/methods/commonParams.md)?